bindings/java/iota-wallet-java/src/main/java/org/iota/Wallet.java

```diff
@@ -0,0 +1,400 @@
+        return new TransactionPayload(responsePayload);
+    }
+
+    public TransactionPayload sendNft(AccountIdentifier accountIdentifier, SendNft method) throws WalletException {
+        JsonObject responsePayload = (JsonObject) callAccountMethod(accountIdentifier, method);
+        return new TransactionPayload(responsePayload);
+    }
+
+    public void setAlias(AccountIdentifier accountIdentifier, SetAlias method) throws WalletException {
+        callAccountMethod(accountIdentifier, method);
+    }
+
+    public TransactionPayload sendOutputs(AccountIdentifier accountIdentifier, SendOutputs method) throws WalletException {
+        JsonObject responsePayload = (JsonObject) callAccountMethod(accountIdentifier, method);
+        return new TransactionPayload(responsePayload);
+    }
+
+    public TransactionPayload signTransactionEssence(AccountIdentifier accountIdentifier, SignTransactionEssence method) throws WalletException {
+        JsonObject responsePayload = (JsonObject) callAccountMethod(accountIdentifier, method);
+        return new TransactionPayload(responsePayload);
+    }
+
+    public TransactionPayload submitAndStoreTransaction(AccountIdentifier accountIdentifier, SubmitAndStoreTransaction method) throws WalletException {
+        JsonObject responsePayload = (JsonObject) callAccountMethod(accountIdentifier, method);
+        return new TransactionPayload(responsePayload);
+    }
+
+    public TransactionPayload claimOutputs(AccountIdentifier accountIdentifier, ClaimOutputs method) throws WalletException {
+        JsonObject responsePayload = (JsonObject) callAccountMethod(accountIdentifier, method);
+        return new TransactionPayload(responsePayload);
+    }
+
+    public void backup(String destination, String password) throws WalletException {
+        JsonObject o = new JsonObject();
+        o.addProperty("destination", destination);
+        o.addProperty("password", password);
+
+        callBaseApi(new ClientCommand("Backup", o));
+    }
+
+    public void changeStrongholdPassword(String currentPassword, String newPassword) throws WalletException {
+        JsonObject o = new JsonObject();
+        o.addProperty("currentPassword", currentPassword);
+        o.addProperty("newPassword", newPassword);
+
+        callBaseApi(new ClientCommand("ChangeStrongholdPassword", o));
+    }
+
+    public void clearStrongholdPassword() throws WalletException {
+        callBaseApi(new ClientCommand("ClearStrongholdPassword"));
+    }
+
+    public boolean isStrongholdPasswordAvailable() throws WalletException {
+        return callBaseApi(new ClientCommand("IsStrongholdPasswordAvailable")).getAsBoolean();
+    }
+
+    public void recoverAccounts(int accountGapLimit, int addressGapLimit, SyncOptions sync_options) throws WalletException {
+        JsonObject o = new JsonObject();
+        o.addProperty("accountGapLimit", accountGapLimit);
```

Comment:
There is now an accountStartIndex

Reply:
are you sure? https://github.com/iotaledger/wallet.rs/blob/develop/src/message_interface/message.rs#L80-L92

Reply:
or do you mean the field in SyncOptions? but in the java binding it exists

Reply:
Ah sorry it's not merged yet, there is just a PR for it